README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
- [ ] Kindly format your README using [the Microverse template](https://github.com/microverseinc/readme-template)